demisto_sdk/commands/validate/initializer.py

```diff
@@ -571,13 +575,23 @@ def get_items_status(
                     statuses_dict[path] = git_status
             elif PACKS_PACK_META_FILE_NAME in path_str:
                 statuses_dict[path] = git_status
-            elif self.is_pack_item(path_str):
+            if (
+                self.is_pack_item(path_str)
+                or find_type(path_str) not in SKIP_RELEASE_NOTES_FOR_TYPES
+            ):
                 metadata_path = self.obtain_metadata_path(path)
                 if metadata_path not in statuses_dict:
                     statuses_dict[metadata_path] = None
             else:
                 statuses_dict[path] = git_status
 
+        all_collected_files = set(statuses_dict.keys())
+        git_only_files = set(file_by_status_dict.keys())
+        related_files = all_collected_files.difference(git_only_files)
+        if related_files:
+            logger.info("Running on related files:")
+            logger.info(f"{[str(path) for path in related_files]}")
```

Comment:
If I'm not wrong - the related files, in this case, is all the files that are not being parsed into graph objects, is this log really helpful?

Reply:
We are still validating them. I also believe they are being parsed into graph objects, just not their previous versions. 
Even if I don't change the pack metadata, I'm still running validate on it & I would like to know about it from the logs. Don't you think?

Reply:
Not sure this log is necessary but we can leave it there if you think it does.